hack/test-cmd.sh

```diff
@@ -142,7 +142,7 @@ os::cmd::expect_success_and_text "openshift ex validate master-config ${MASTER_C
 os::cmd::expect_success_and_text "openshift ex validate node-config ${NODE_CONFIG_DIR}/node-config.yaml" 'SUCCESS'
 # breaking the config fails the validation check
 cp ${MASTER_CONFIG_DIR}/master-config.yaml ${BASETMPDIR}/master-config-broken.yaml
-os::util::sed '7,12d' ${BASETMPDIR}/master-config-broken.yaml
+echo "kubernetesMasterConfig: {}" >> ${BASETMPDIR}/master-config-broken.yaml
```

Comment:
Previously, there was an `apiVersion` and `kind` that allowed for proper decoding.  I would expect this change to break the decoding path and fail earlier than intended.  I think that this change actually skips the bit where we're validating the config because I don't see how it could construct the object correctly.